client/Contexts/Introspect/mock/expectations-basic.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import { Expectations } from '../ExpectationTypes';
+
+export default {
+  Other: {
+    type: 'Other',
+    fields: {
+      names: '[String]',
+      size: 'Int', // TODO how do we handle args?
+      roughSize: 'TShirtSize',
+      requiredRoughSize: 'TShirtSize!',
+      possibleRoughSizes: '[TShirtSize]',
+      requiredNames: '[String]!',
+      nonNullNames: '[String!]',
+      nonNullRequiredNames: '[String!]!',
+      topics: '[Topic]',
+    },
```

Comment:
Is this just to illustrate a type can reference others? If so, would doc be a suitable place for it? Might save on the TODO too?

Reply:
No, it's mock data used to test the introspector. It's important to validate that the instrospector works with all different types in GraphQL IMO.

Reply:
no, it's mock data used to validate the introspector

Reply:
The TODO is valid, but we can just make an issue for it - I don't think it's a common case.